build/wikibase-lts/CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+## 1.0.1 (2025-01-06)
+
+
+### 🏡 Chore
+
+- bump mediawiki to 1.42.3, bump extensions
+
+## (from **wikibase@3.0.1** (2024-10-09))
```

Comment:
I am unsure whether we really want to include this information. Going forward, NX wouldn't generate it either. I think I would prefer just the list of changes since wikibase-lts@1.0.0